src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1233,6 +1260,50 @@ private void UpdateEditor(float delta)
         {
             CurrentOrganelleCost = 0;
         }
+
+        // Apply camera movement
+        camera.ObjectToFollow.Translation += new Vector3(cameraMovement.x, 0, cameraMovement.y) * delta;
+    }
+
+    private void HandleMovement()
+    {
+        cameraMovement = Vector2.Zero;
+        if (mousePanningStart == null)
+        {
+            if (Input.IsActionPressed("e_pan_left"))
+            {
+                cameraMovement += Vector2.Left;
+            }
+
+            if (Input.IsActionPressed("e_pan_right"))
+            {
+                cameraMovement += Vector2.Right;
+            }
+
+            if (Input.IsActionPressed("e_pan_down"))
+            {
+                cameraMovement += Vector2.Down;
+            }
+
+            if (Input.IsActionPressed("e_pan_up"))
+            {
+                cameraMovement += Vector2.Up;
+            }
+
+            cameraMovement = cameraMovement.Normalized() * camera.CameraHeight;
+        }
+        else
+        {
+            // ReSharper disable once PossibleInvalidOperationException
+            var direction = mousePanningStart.Value - camera.CursorWorldPos;
+            camera.ObjectToFollow.Translation += direction;
+        }
+    }
+
+    private void ResetCamera()
+    {
+        camera.ResetPosition();
+        organelleRot = 0;
```

Comment:
Why is the rotation of the to be placed organelle reset here? That doesn't seem very related to the camera.

Reply:
oh, I thought that was meant to rotate the camera itself and was unfinished, not the placed organelle! I never knew you could rotate them :D